src/libYARP_dev/src/devices/Rangefinder2DClient/Rangefinder2DClient.cpp

```diff
@@ -246,9 +246,9 @@ bool yarp::dev::Rangefinder2DClient::open(yarp::os::Searchable &config)
 
     if (b_canOpenTransformClient)
     {
-        yarp::dev::IFrameTransform* iTrf = 0;
+        yarp::dev::IFrameTransform* iTrf = YARP_NULLPTR;
         drv->view(iTrf);
-        if (iTrf != 0)
+        if (iTrf == YARP_NULLPTR)
```

Comment:
I would go for `if(!iTrf)`. 
Anyhow, how it is possible that this code ever worked? Before merging I think the original author @randaz81 should take a look a this.

Reply:
LGTM, why should not work?

Reply:
Modified in https://github.com/robotology/yarp/pull/1387/commits/d6e0a191dc8e18d177aa314512647fbc7fd03d0c. Don't know about previous behaviour